cloudfour/cloudfour.com-patterns
# Retain supposedly "useless" SVG stroke and fill attributes

## Overview

I was writing an issue related to icon design updates and noticed that the `check` icon was invisible in Storybook.

This was because `check` was most recently updated and is the only icon that does not define `stroke="currentColor"`, since that attribute is no longer necessary with `o-icon`. But `svgo` interpreted that as "oh, this has no stroke" and removed _all_ `stroke-` attributes.

This tweaks the SVGO settings (which are, happily, shared by both the Twig conversion and Storybook) to stop doing that.

## Screenshots

### Before

<img width="174" alt="Screen Shot 2020-04-24 at 4 10 31 PM" src="https://user-images.githubusercontent.com/69633/80263754-8971f100-8646-11ea-99c7-b89548a859c4.png">

### After

<img width="180" alt="Screen Shot 2020-04-24 at 4 10 18 PM" src="https://user-images.githubusercontent.com/69633/80263759-8d9e0e80-8646-11ea-8113-111ec723df5b.png">

## Testing

1. View Design ▸ Icons page in deploy preview.
1. Confirm the `check` icon is visible.
1. View Components ▸ Checkbox page in deploy preview.
1. Confirm icon is still visible in checked boxes.
